packages/DApp/src/components/NotificationsList.tsx

```diff
@@ -14,6 +14,7 @@ export function NotificationsList() {
       {notifications.map((notification) => {
         if ('receipt' in notification) {
           return notification.receipt.logs.map((log) => {
+            // this needs to be updated so it takes into account also interface of featuredVotingContract
             const parsedLog = votingContract.interface.parseLog(log)
```

Comment:
Here is the issue with NotificationList, it accept list of notifications which can be emmited by both contracts (voting and featuredVoting), but there is currently no way how to differentiate notification item and therefore parse it by parser with correct interface. If notification item is parsed by wrong parser it throws an exception and crashes whole page.